screens/Map.js

```diff
@@ -1,81 +1,82 @@
 /* eslint-disable no-unused-vars */
-import React, { useEffect } from "react";
-import { StyleSheet, AsyncStorage } from "react-native";
+import React from "react";
+import { StyleSheet, Keyboard  } from "react-native";
 import MapView, { PROVIDER_GOOGLE } from "react-native-maps";
-import { BuildingHighlight } from "../components/BuildingHighlight";
+import BuildingHighlight  from "../components/BuildingHighlight";
 import { BuildingIdentification } from "../components/BuildingIdentification";
 import { BottomMenu } from "../components/BottomMenu";
 import { CurrentBuildingLocation } from "../components/CurrentBuildingLocation";
```

Comment:
<img width="751" alt="Screen Shot 2020-04-18 at 1 22 17 PM" src="https://user-images.githubusercontent.com/35850933/79644544-a2c2fb00-8177-11ea-8a9a-01a590e5ea34.png">

Reply:
I'm fixing this component, it will be there!